boba-bindings/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest AS builder
+FROM us-docker.pkg.dev/boba-392114/bobanetwork-tools-artifacts/images/ci-builder:latest AS builder
```

Comment:
Probably fine, as it's not actually upstream, but why not use `${GCP_PROJECT_ID/${GCP_ARTIFACT_REPOSITORY}` as in the other dockerfiles, that way if we end up having to switch project IDs or repositories, this won't break?